### ISSUE TITLE
Add

### ISSUE THREAD

**Issue by luisdev** [moneymanagerex/moneymanagerex#6199]
MMEX version:
 - [ ] 1.6.4
 - [x] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the feature request

Please add a Description column to the new Organise Tags screen. It will be really nice to be able to add a description to each tag that you create. It helps keep track of the reasons why you created each tag and how you intend to use each tag.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
Open the Organise Tags screen from the Tools menu. Currently it just shows a list of all the tag names that have already been defined.

Expected result:
Open the Organise Tags screen from the Tools menu and you'll see two columns. The first column is the tag name and the second column is a description column.

![Organise Tags Screen Description Column](https://github.com/moneymanagerex/moneymanagerex/assets/1124791/a073b539-dd0f-4696-90d3-57525722fb9c)

See attached screenshot of the proposed screen.

# Additional information

N/A

**Comment by whalley** [comment]
For completeness maybe we should consider supporting descriptions against Categories and Payees.

**Comment by luisdev** [comment]
> For completeness maybe we should consider supporting descriptions against Categories.

Yes, that would be great.

**Comment by vomikan** [comment]
> For completeness maybe we should consider supporting descriptions against Categories.

You can continue this thought and think about the description of payees.

**Comment by whalley** [comment]
I did, but realised we already had a number of 'description' fields associated with the payee, including a notes field.

**Comment by vomikan** [comment]
> I did, but realised we already had a number of 'description' fields associated with the payee, including a notes field.

These are slightly different places. If you add a description to payee, it will be visible only in the configurator. This is information about payee, but not about the transaction.
As for me, the very name of payee may indicate its purpose. Also with categories. Tags can also be made to speak for themselves.

**Comment by luisdev** [comment]
> These are slightly different places. If you add a description to payee, it will be visible only in the configurator. This is information about payee, but not about the transaction. As for me, the very name of payee may indicate its purpose. Also with categories. Tags can also be made to speak for themselves.

For me the priority is for the Tags description column. It will make my migration to MMX from Quicken so much easier.

**Comment by luisdev** [comment]
This is not Stale for me! It would still be nice to have it.
